content/docs/plugins/fields.md

```diff
@@ -129,6 +129,8 @@ The `name` property should be updated to reflect the different path:
 }
 ```
 
+> Checkout [the final-form docs]() for a more detailed look at how the `name` field works.
```

Comment:
```suggestion
> Check out [the final-form docs](https://final-form.org/docs/final-form/field-names) for a more detailed look at how the `name` field works.
```

@ncphillips Is that the right link?

Reply:
Yes. I forgot to hit save on the wysiwyg link ui

Reply:
ENTER will also save the link.